src/nodes/figure.ts

```diff
@@ -79,21 +80,23 @@ export const toTex = createLatexStatement(
   (state, node) => {
     const { numbered, id } = node.attrs as Attrs;
     const localId = state.options.localizeId?.(id ?? '') ?? id;
-    const star = numbered ? '' : '*';
     // TODO: Based on align attr
     // may have to modify string returned by state.renderContent(node);
     // https://tex.stackexchange.com/questions/91566/syntax-similar-to-centering-for-right-and-left
     state.write('\\centering');
     const captionFirst = node.firstChild?.type.name === nodeNames.figcaption;
     if (localId && captionFirst) {
       state.ensureNewLine();
-      state.write(`\\label${star}{${localId}}`);
```

Comment:
I have removed `\label*{}` which I don't think is valid latex.

Reply:
Ah correct!